mechanism2d/robot.py

```diff
@@ -0,0 +1,56 @@
+import wpilib
+
+"""
+This sample program shows how to use Mechanism2d - a visual representation of arms, elevators,
+and other mechanisms on dashboards; driven by a node-based API.
+
+Ligaments are based on other ligaments or roots, and roots are contained in the base
+Mechanism2d object.
+"""
```

Comment:
This string is not a docstring, as it isn't the first statement.
```suggestion
"""
This sample program shows how to use Mechanism2d - a visual representation of arms, elevators,
and other mechanisms on dashboards; driven by a node-based API.

Ligaments are based on other ligaments or roots, and roots are contained in the base
Mechanism2d object.
"""

import wpilib
```